.github/workflows/rust.yml

```diff
@@ -42,26 +42,10 @@ jobs:
       run: cargo fmt --check
     # Build and run the tests
     - name: Build and run tests
-      run: cargo test --verbose --release
-    # Run our non-interactive examples and assert the complete without error
-    - name: Verify examples (amm)
```

Comment:
We're no longer running our examples?

Reply:
The `test` step runs all tests in the workspace, including examples. I just made sure each example had a `#[test]` testing `main`.